From 1d4f9e1ca5a27426da4dcbd3caee1e38b5d272aa Mon Sep 17 00:00:00 2001
From: Justin Lecher <jlec@gentoo.org>
Date: Tue, 27 Dec 2016 13:17:27 +0000
Subject: [PATCH] Use bash internal functionalities for grc detection

which is an external tool, where as type is a bash internal funciton
and always available

Signed-off-by: Justin Lecher <jlec@gentoo.org>
---
 grc.bashrc | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/grc.bashrc b/grc.bashrc
index e9eba08..54e570d 100644
--- a/grc.bashrc
+++ b/grc.bashrc
@@ -1,6 +1,5 @@
-GRC=`which grc`
-if [ "$TERM" != dumb ] && [ -n "$GRC" ]
-then
+GRC="$(type -p grc)"
+if [ "$TERM" != dumb ] && [ -n "$GRC" ]; then
     alias colourify="$GRC -es --colour=auto"
     alias configure='colourify ./configure'
     alias diff='colourify diff'
